pypoptools/pypoptesting/framework/pop_util.py

```diff
@@ -0,0 +1,64 @@
+from .node import Node
+from .sync_util import wait_for_block_height
+
+KEYSTONE_INTERVAL = 5
```

Comment:
remove. this is altchain-specific parameter, and will be fetched in tests via getpopparams

Reply:
removed